gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/FormatHelper.java

```diff
@@ -384,4 +384,13 @@ public static String decode(final String s) {
 	public static String htmlEscape(String input){
 		return HtmlUtils.htmlEscape(input, StandardCharsets.UTF_8.name());
 	}
+	
+	/**
+	* Turn HTML character references into special characters. Handles complete character set defined in HTML 4.01 recommendation.
+	* Unescapes all entities references (e.g. &lt;) to their special characters.
+	* Reference: http://www.w3.org/TR/html4/sgml/entities.html
+	 */
```

Comment:
You don't need the HtmlUtils javadoc here. It will get out of sync with the original and people can just read it there, anyway.

Reply:
Done